### PR TITLE
[Feature] Add reading progress bar

### PR DESCRIPTION
**Pull request by dmitry-sidorov**
@lubien What's up!

I've implemented a progress bar for this beautiful and useful LiveView tutorial.
Review it, please.
This PR is related to https://github.com/adopt-liveview/adopt-liveview/issues/47

Best regards 😄 

https://github.com/user-attachments/assets/82ca45f2-fb80-44b3-8b0d-97990efcdaab

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 To start your Phoenix server:
 
+  * Create database container `docker run --name curso_dev -p 5432:5432 -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=postgres -d postgres`
```
Comment (lubien) [inline README.md:5]: I dont think we need that!

File: lib/curso_web/components/core_components.ex [hunk 2]
```diff
@@ -1421,4 +1417,24 @@ defmodule CursoWeb.CoreComponents do
...
     for {^field, {msg, opts}} <- errors, do: translate_error({msg, opts})
   end
+
+  @doc """
+  Shows progress bar in the page header.
+  """
+  attr :progress, :integer, default: 0, doc: "Page reading progress"
+
+  def progress_bar(assigns) do
+    assigns |> dbg()
```
Comment (lubien) [inline lib/curso_web/components/core_components.ex:1427]: Can you remove this line?

File: lib/curso_web/live/guide_live.ex [hunk 3]
```diff
@@ -60,6 +61,12 @@ defmodule CursoWeb.GuideLive do
     "#{page.title} · #{page.section}"
   end
 
+  def handle_event("reading_progress", page_progress, socket) do
+    socket = assign(socket, page_progress: page_progress)
+    {:reply, %{}, socket}
+  end
+
+  @spec render(any()) :: Phoenix.LiveView.Rendered.t()
```
Comment (lubien) [inline lib/curso_web/live/guide_live.ex:69]: Can you remove this line?

**Review by lubien — CHANGES_REQUESTED** [review]
Great stuff! Asked some minor changes

File: README.md (hunk 1 above)
Comment (dmitry-sidorov) [reply README.md]: Removed!

File: lib/curso_web/live/guide_live.ex (hunk 3 above)
Comment (dmitry-sidorov) [reply lib/curso_web/live/guide_live.ex]: Autoformatter, I guess

**Comment by dmitry-sidorov** [comment]
@lubien up

**Comment by dmitry-sidorov** [comment]
@lubien Hey, man! I hope you're doing well! Could you, please, review these tiny fixes and we'll be able to merge it and improve this cool project?

**Review by lubien — APPROVED** [review]

**Pull request merged**